app/models/concerns/orchestration/tftp.rb

```diff
@@ -146,7 +157,11 @@ def queue_tftp
   end
 
   def queue_tftp_create
+    pxe_kind = host.operatingsystem.pxe_loader_kind(host)
     host.operatingsystem.template_kinds.each do |kind|
+      if kind.to_s == "PXEGrub2" && pxe_kind.to_s == "PXEGrub2TargetOS"
```

Comment:
Do you have an idea how to implement this in a more generic way? Or should we really have hard-coded this.

Reply:
Deprecated with our new approach